benchmarks/README.md

```diff
@@ -80,6 +80,67 @@ using `pyarrow` dtypes).
 | Dependencies._update_media()                    |    0.087 |    0.086 |     0.145 |
 | Dependencies._update_media_version(10000 files) |    0.011 |    0.011 |     0.020 |
 
+## audb.Dependencies methods using polars
+
+Pyarrow was further  compared to a reimplenentation  of the dependency
+module  by  monkeypatching using  the  polars  dataframe engine.   All
+methods of `audb.Dependencies` were reimplemented using `polars`.
+
+* CPU: 12th Gen Intel Core i7-1255U
+* RAM: 15.66 GB
+* Hard Drive: KBG5AZNT1T02 LA KIOXIA
+* Linux: Ubuntu 22.04.4 LTS
+* Python 3.11.9
+
+To run the benchmark execute:
+
+```bash
+$ python benchmark-dependencies-methods-polars.py
+```
+
+The data were compared to
+the results from `benchmark-dependencies-methods.py` using
+
+```bash
+python compare_dependency_methods_polars.py
+```
+Both steps require that `benchmark-dependencies-methods.py`
+has been run previously in order to create the test data
+and results.
+
```

Comment:
Can you add a sentence here which of the three previous methods from the table before is referred to by `pandas`. I guess `pyarrow`?

I would stick with the name `pandas` in the table as this is what we are comparing against.

Reply:
I have added a sentence that mentions that the `pyarrow` column is the relevant comparison in the previous table.

Reply:
Thanks.